test/inductor/test_codecache.py

```diff
@@ -1,9 +1,21 @@
 # Owner(s): ["module: inductor"]
```

Comment:
Just based on the name, this seemed like the right file to place any new "codecache" unit tests. Lemme know if there's a different convention, however. I was a little confused about the existing stuff in this file because AFAICT there were no unit tests running.

@ani300, you might notice that I didn't preserve any of your previous tests. TBH, I just couldn't figure out what they were doing and what I have here seemed more straightforward (to me). Please help me out if I should actually be carrying over some of the tests you'd written and I'll gladly bring them back.

Reply:
My tests were only testing the behavior of compile model, dynamo reset, compile model again and see if we hit the cache if we expect to. I was missing all the hashing tests that you've added and kwargs ordering etc.